theforeman.org/pipelines/lib/git.groovy

```diff
@@ -11,8 +11,13 @@ def ghprb_git_checkout() {
     ]
 }
 
+def git_hash(ref = 'HEAD') {
+    return sh(script: "git rev-parse ${ref}", returnStdout: true).trim()
+}
+
 def archive_git_hash(ref = 'HEAD') {
-    def hash = sh(script: "git rev-parse ${ref} | tee commit", returnStdout: true).trim()
+    def hash = git_hash(ref)
+    writeFile('commit', dash)
```

Comment:
```suggestion
    writeFile('commit', hash)
```

Reply:
huh, wth was I thinking

Reply:
dash is the default `/bin/sh` on Debian, so idk?